ui/page/dcrdex/dcrdex_page.go

```diff
@@ -63,6 +70,9 @@ func (pg *DEXPage) OnNavigatedTo() {
 // eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *DEXPage) Layout(gtx C) D {
+	if !pg.WL.AssetsManager.IsDexFirstVisit() {
+		return components.UniformPadding(gtx, pg.splashPage)
+	}
```

Comment:
Uhm, you show the splash page if it is not the first visit? Should this be reversed?